pages/06_embedding_non_view_content.md

```diff
@@ -2,22 +2,64 @@
 title: Embedding Non-View Content
 ---
 
-Almost all embedded deployments of Tableau embed pre-built dashboards. But many embed other components of Tableau Server such as:
+Almost all embedded deployments of Tableau embed pre-built dashboards. But some embed other components of Tableau such as:
 
-* Web Authoring screens
+* web authoring views
 * The Tableau Server UI (embedded for integration and single sign-on reasons)
```

Comment:
We should add: "Not supported by Tableau"

Reply:
@illonage  - you mean add the "Not supported by Tableau" to the second bullet item?  That is, the Tableau Server UI?

Reply:
yes correct, just for bullet 2